docs/azure/includes/dotnet-all.md

```diff
@@ -27,7 +27,7 @@
 | Event Hubs - Schema Registry | NuGet [1.0.0-beta.2](https://www.nuget.org/packages/Azure.Data.SchemaRegistry/1.0.0-beta.2) | [docs](/dotnet/api/overview/azure/Data.SchemaRegistry-readme-pre/) | GitHub [1.0.0-beta.2](https://github.com/Azure/azure-sdk-for-net/tree/Azure.Data.SchemaRegistry_1.0.0-beta.2/sdk/schemaregistry/Azure.Data.SchemaRegistry/) |
 | Form Recognizer | NuGet [3.0.1](https://www.nuget.org/packages/Azure.AI.FormRecognizer/3.0.1)<br>NuGet [3.1.0-beta.4](https://www.nuget.org/packages/Azure.AI.FormRecognizer/3.1.0-beta.4) | [docs](/dotnet/api/overview/azure/AI.FormRecognizer-readme/) | GitHub [3.0.1](https://github.com/Azure/azure-sdk-for-net/tree/Azure.AI.FormRecognizer_3.0.1/sdk/formrecognizer/Azure.AI.FormRecognizer/)<br>GitHub [3.1.0-beta.4](https://github.com/Azure/azure-sdk-for-net/tree/Azure.AI.FormRecognizer_3.1.0-beta.4/sdk/formrecognizer/Azure.AI.FormRecognizer/) |
 | Identity | NuGet [1.3.0](https://www.nuget.org/packages/Azure.Identity/1.3.0)<br>NuGet [1.4.0-beta.5](https://www.nuget.org/packages/Azure.Identity/1.4.0-beta.5) | [docs](/dotnet/api/overview/azure/Identity-readme/) | GitHub [1.3.0](https://github.com/Azure/azure-sdk-for-net/tree/Azure.Identity_1.3.0/sdk/identity/Azure.Identity/)<br>GitHub [1.4.0-beta.5](https://github.com/Azure/azure-sdk-for-net/tree/Azure.Identity_1.4.0-beta.5/sdk/identity/Azure.Identity/) |
-| IoT Device Update | NuGet [1.0.0-beta.2](https://www.nuget.org/packages/Azure.IoT.DeviceUpdate/1.0.0-beta.2) | [docs](/dotnet/api/overview/azure/IoT.DeviceUpdate-readme-pre/) | GitHub [1.0.0-beta.2](https://github.com/Azure/azure-sdk-for-net/tree/Azure.IoT.DeviceUpdate_item.Version/sdk/deviceupdate/Azure.Iot.DeviceUpdate/) |
+| IoT Device Update | NuGet [1.0.0-beta.2](https://www.nuget.org/packages/Azure.IoT.DeviceUpdate/1.0.0-beta.2) | [docs](/dotnet/api/overview/azure/IoT.DeviceUpdate-readme-pre/) | GitHub [1.0.0-beta.2](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/deviceupdate/Azure.IoT.DeviceUpdate) |
```

Comment:
⚠️ I believe this will get overwritten with the next bot PR. The fix should be somewhere else. (Probably https://github.com/Azure/azure-sdk/blob/master/_data/releases/latest/dotnet-packages.csv?)